lib/config/definitions.ts

```diff
@@ -281,11 +281,11 @@ const options: RenovateOptions[] = [
   {
     name: 'binarySource',
     description:
-      'Where to source binaries like `npm` and `yarn` from, choices are `auto`, `global` and `docker`',
+      'Controls whether third party tools like npm or gradle are called directly, or via Docker sidecar containers',
```

Comment:
```suggestion
      'Controls whether third party tools like npm or gradle are called directly, or via Docker sidecar containers.',
```

Add missing punctuation. 😄 🙃